README.md

```diff
@@ -1,80 +1,87 @@
-#kdb-rest-service
+# kdb-rest-service
 
 This is an open source rest service written in java which can be used to connect to an instance of kdb using json. The rest service can provide a single query to run or call a function predefined on the instance of kdb.
```

Comment:
I'd remove "This is" and start with "An open source..."; the "this is" adds nothing IMO and I think it's advantageous to keep the description as brief as possible

possibly also capitalise JSON & REST given they are abbreviations?